.nuke/build.schema.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Build Schema",
+  "$ref": "#/definitions/build",
+  "definitions": {
+    "build": {
+      "type": "object",
+      "properties": {
+        "Configuration": {
+          "type": "string",
+          "description": "Configuration to build - Default is 'Debug' (local) or 'Release' (server)"
```

Comment:
nit
```suggestion
          "description": "Configuration to build - Default is 'Debug'. Possible Values: 'Debug' (local) or 'Release' (server)"
```

Reply:
This originates [here](https://github.com/dotnet/Silk.NET/blob/073e45d090fc59f045d2e606fec6b9f18e8f48b4/build/nuke/Build.Support.cs#L76) but I can't be bothered to open up the solution just to add 3 words very few people will ever see  and even fewer people will misunderstand...